src/app/main/component/ubs/components/ubs-submit-order/ubs-submit-order.component.ts

```diff
@@ -63,6 +71,19 @@ export class UBSSubmitOrderComponent extends FormBaseComponent implements OnInit
   }
 
   redirectToOrder() {
-    document.location.href = this.shareFormService.orderUrl;
+    this.loadingAnim = true;
+    this.orderService
+      .getOrderUrl()
+      .pipe(takeUntil(this.destroy))
+      .subscribe(
+        (fondyUrl) => {
+          this.shareFormService.orderUrl = fondyUrl.toString();
```

Comment:
will this not be string by default ?

Reply:
I will investigate this.

Reply:
Yea, it's a string, but I understood why there was a .toString() method.
![image](https://user-images.githubusercontent.com/43093994/123388347-5bb56e80-d5a1-11eb-93a5-545fdf0f1bff.png)